src/utils/helper.ts

```diff
@@ -0,0 +1,9 @@
+import { Cart } from "../types/cart.interface";
+
+
+export const getTotalPriceOfCart = (cart: Cart) => {
+    let totalPrice = 0;
+    cart.articles.forEach(i => totalPrice += (i.price * i.CartArticle!.quantity))
+ 
+    return totalPrice;
+   }
```

Comment:
use proper array prototype methods that will make code much cleaner and faster. array.prototype.reduce in this case